app/controllers/conference_registrations_controller.rb

```diff
@@ -89,9 +87,9 @@ def destroy
       redirect_to root_path,
                   notice: "You are not registered for #{@conference.title} anymore!"
     else
-      redirect_to root_path,
-                  error: "Could not update your registration for #{@conference.title}: "\
-                  "#{@registration.errors.full_messages.join('. ')}."
+      redirect_to conference_conference_registrations_path(@conference.short_title),
```

Comment:
This is a singular resource (http://guides.rubyonrails.org/routing.html, 2.5 Singular Resources):

```
A singular resourceful route generates these helpers:

new_geocoder_path returns /geocoder/new
edit_geocoder_path returns /geocoder/edit
geocoder_path returns /geocoder
```

So `conference_conference_registration_path` for **show** action.